jdbc/pom.xml

```diff
@@ -74,8 +74,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>2.7.2</version>
-      <scope>provided</scope>
+      <version>${hadoop.common.version}</version>
     </dependency>
```

Comment:
I tried, but I guess it require x.y.z https://repo.maven.apache.org/maven2/org/apache/hadoop/hadoop-common/